### PR TITLE
Incorrect information regarding instant-meilisearch

### LINKED ISSUE #1603: Incorrect information regarding instant-meilisearch
In [Step 5 of the quick start guide](https://docs.meilisearch.com/learn/getting_started/quick_start.html#step-5-front-end-integration), instant-meilisearch is described as "a developer tool that generates all the components needed to start searching.". That's not true, it's [instantSearch](https://github.com/algolia/instantsearch.js) powered by Algolia that generates the tools, instant-meilisearch only connects those tools with Meilisearch (if I am not wrong)

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #1603

### CODE REVIEW

**Review by bidoubiwa — COMMENTED** [review]
Do we want to say that instantsearch is powered by algolia? Not that I'm against it, but, for example, facebook created `React` but we do not mention them when talking about react plugins!

**Comment by maryamsulemani97** [comment]
I saw it in the instant-meilisearch README so I added it, but I can remove it

File: learn/getting_started/quick_start.md [hunk 1]
```diff
@@ -271,7 +271,7 @@ If you have multiple indexes, you can switch between them using the indexes drop
 
 ## Front-end integration
 
-The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a developer tool that generates all the components needed to start searching.
+The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and the open-source [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch, powered by Algolia, generates all the components needed to start searching.
```
Comment (maryamsulemani97) [inline learn/getting_started/quick_start.md:274]:
```suggestion
The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and the open-source [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch generates all the components needed to start searching.
```

File: learn/getting_started/quick_start.md [hunk 2]
```diff
@@ -271,7 +271,7 @@ If you have multiple indexes, you can switch between them using the indexes drop
 
 ## Front-end integration
 
-The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a developer tool that generates all the components needed to start searching.
+The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and the open-source [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch generates all the components needed to start searching.
```
Comment (bidoubiwa) [inline learn/getting_started/quick_start.md:274]:
```suggestion
The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch is the tool that renders all the components needed to start searching.
```

I think I will update the readme of instant meilisearch accordingly :')

**Comment by bidoubiwa** [comment]
@curquiza good for me

**Review by bidoubiwa — DISMISSED** [review]
LGTM 🔥

File: learn/getting_started/quick_start.md [hunk 3]
```diff
@@ -271,7 +271,7 @@ If you have multiple indexes, you can switch between them using the indexes drop
 
 ## Front-end integration
 
-The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a developer tool that generates all the components needed to start searching.
+The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch, powered by Algolia, is the tool that renders all the components needed to start searching.
```
Comment (curquiza) [inline learn/getting_started/quick_start.md:274]:
```suggestion
The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch, an open-source project powered by Algolia, is the tool that renders all the components needed to start searching.
```

**Review by curquiza — DISMISSED** [review]

**Review by bidoubiwa — DISMISSED** [review]
LGTM 🔥

**Review by guimachiavelli — APPROVED** [review]
I'm pre-approving this with one important comment.

We say InstantSearch is powered by Algolia. "powered by" implies that "InstantSearch" is built on top of Algolia/has Algolia as a dependency. Is that correct? For example, a blog can be "powered by Wordpress" and  Next.js is "powered by React".

**Comment by bidoubiwa** [comment]
> We say InstantSearch is powered by Algolia. "powered by" implies that "InstantSearch" is built on top of Algolia/has Algolia as a dependency. Is that correct? For example, a blog can be "powered by Wordpress" and Next.js is "powered by React".

InstantSearch is not built on top of Algolia! For me `powered by` is the creator/maintainer, but I may have been wrong all that time :')

**Comment by guimachiavelli** [comment]
There is some ambiguity with "powered by", IMHO. I agree, depending on the context it can be the creator/maintainer, but it can also imply a tighter coupling like "depends on".

To avoid confusion, I suggest we change from "powered by" to "developed by".

File: learn/getting_started/quick_start.md [hunk 4]
```diff
@@ -271,7 +271,7 @@ If you have multiple indexes, you can switch between them using the indexes drop
 
 ## Front-end integration
 
-The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a developer tool that generates all the components needed to start searching.
+The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch, an open-source project powered by Algolia, is the tool that renders all the components needed to start searching.
```
Comment (maryamsulemani97) [inline learn/getting_started/quick_start.md:274]:
```suggestion
The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch, an open-source project developed by Algolia, is the tool that renders all the components needed to start searching.
```

**Review by bidoubiwa — APPROVED** [review]

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**